quasar.config.js

```diff
@@ -207,15 +207,19 @@ module.exports = configure(function (/* ctx */) {
         },
         linux: {
           target: "AppImage",
+          category: "Utility",
         },
         win: {
           target: "nsis",
         },
         nsis: {
+          oneClick: false,
```

Comment:
Unfortunately, according to electron-builder's [documentation](https://www.electron.build/configuration/nsis), `deleteAppDataOnUninstall` is for one-click installer only.

> `deleteAppDataOnUninstall` = false Boolean - one-click installer only. Whether to delete app data on uninstall.

I like the idea that user can choose their install path, it is a feature I want as well. To resolve this conflict, I need to do some research on the uninstall-script for all platforms. 

An easier way is to remove the `deleteAppDataOnUninstall` option, I will write some detail steps about how to remove user data on documentation page and let users to figure out themselves.